app/services/idv/vendor_result.rb

```diff
@@ -14,8 +14,8 @@ def self.new_from_json(json)
     def initialize(success: nil, errors: nil, reasons: nil, session_id: nil,
                    normalized_applicant: nil)
       @success = success
-      @errors = errors
-      @reasons = reasons
+      @errors = errors || {}
```

Comment:
If this need to be an empty hash by default, why not do it in the keyword arguments? Same with `reasons` below.